ElasticsearchReportHelper.php

```diff
@@ -381,7 +381,8 @@ public static function download(array $options) {
     if (!empty($options['columnsTemplate']) && is_array($options['columnsTemplate'])) {
       $availableColTypes = array(
         "default" => lang::get("Standard download format"),
-        "easy-download" => lang::get("Backward-compatible format")
+        "easy-download" => lang::get("Backward-compatible format"),
+        "mapmate" => lang::get("Mapmate-compatible format")
```

Comment:
Being really fussy (!) but Drupal coding standards require a comma after last entry of a multi-line array.

Reply:
I've fixed that @johnvanbreda.